packages/sanity/src/structure/panes/documentList/useDocumentSheetColumns.tsx

```diff
@@ -156,17 +160,19 @@ export function useDocumentSheetColumns(documentSchemaType?: ObjectSchemaType) {
     () =>
       flatColumns(columns).reduce<[VisibilityState, number]>(
         ([accCols, countAllowedVisible], column) => {
+          const visibilityKey = String(column.id)
```

Comment:
Column id is possibly undefined according to the tanstack column definition and accessor functions, thinking on throwing an error if it doesn't have a `column.id`
We are controlling this, but in case something changes in the future it will be simpler to identify the problem if for some case the id is missing.

Reply:
- [ ] throw if there's not an ID on a column